plato/trainers/basic.py

```diff
@@ -110,8 +111,8 @@ def load_model(self, filename=None, location=None):
 
     def simulate_sleep_time(self):
         """Simulate client's speed by putting it to sleep."""
-        if not (hasattr(Config().clients, 'sleep_simulation')
-                and Config().clients.sleep_simulation):
+        if hasattr(Config().clients,
+                   'sleep_simulation') and Config().clients.sleep_simulation:
```

Comment:
Here, the original code was correct. `sleep_simulation` equals `true` implies that the entire training session will not actually going to sleep, and instead simulate it. In this case, we are not randomly generating the number of seconds.

Reply:
Oh I see. It is now updated. Thanks!